### PR TITLE
prepare for v1.0.0

### PR DESCRIPTION
**Pull request by magicaljellybeans**
- Updated Changelog
- Updated setup.py
- Updated documentation with upgrade instructions
- Updated contributing documentation with version requirements

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +68,18 @@ Include the Javascript in your base template
 ```html
   <script type="text/javascript" src="{% static 'wagtailnhsukfrontend/js/nhsuk-4.1.0.min.js' %}" defer></script>
 ```
+## Upgrading
```
Comment (mikemonteith) [inline README.md:71]: I don't think this "upgrading" is necessary in the readme. Having notes in the changelog should be enough.
Maybe a small section that says "If you are using v0 and wish to upgrade to v1, see the changelog" with a link to the changelog "upgrade considerations" notes.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,10 +1,15 @@
 # CHANGELOG
 
-## Master
+## v1.0.0
 
+### Upgrade Considerations
+- Panel Block, Panel List, Grey Panel Block, Promo Block, Promo Group Block have been removed from the plugin, so references to these in your project will need to be removed in order to upgrade. Ensure to make migrations after doing so.
```
Comment (mikemonteith) [inline CHANGELOG.md:6]: We should probably refer to these blocks by their python class name (`PanelBlock`, `PanelList` etc.)

**Review by mikemonteith — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,10 +1,15 @@
 # CHANGELOG
 
-## Master
+## v1.0.0
 
+### Upgrade Considerations
+- `PanelBlock`, `PanelList`, `GreyPanelBlock`, `PromoBlock`, `PromoGroupBlock` have been removed from the plugin, so references to these in your project will need to be removed in order to upgrade. Ensure to make migrations after doing so.
```
[the remark this answers is not in the document]
Comment (dragon-dxw) [reply CHANGELOG.md:6]: I'd certainly appreciate advice on the recommended way of transferring content currently in deprecated blocks: which fields should we remap to which blocks for the best user experience?

[the remark this answers is not in the document]
Comment (magicaljellybeans) [reply CHANGELOG.md]: @dragon-dxw I've updated the changelog with some more information - does this tell you everything you need to know?

[the remark this answers is not in the document]
Comment (dragon-dxw) [reply CHANGELOG.md]: Haven't used it in anger yet, but that looks amazing and exactly like the shape of the information I was hoping to receive. I'm pretty confident (misplaced or not) that I can make that work. I'll dig into it when I get a chance and let you know how I get on. 

Thank you!

[the remark this answers is not in the document]
Comment (mikemonteith) [reply CHANGELOG.md]: Thanks @dragon-dxw please do get in touch with us by opening an issue if you want any help with the upgrade or things aren't clear.

**Review by mikemonteith — APPROVED** [review]

**Pull request merged**